### DISCUSSION TITLE
GHelper Power limits causing CPU to downclock on 0.44Ghz on startup

### DISCUSSION THREAD

**Discussion by Doughkey** [seerge/g-helper#3496] [General]
Concern: GHelper Power limits are causing laptop to lag and become sluggish on startup. Upon checking, GHelper starting up causes CPU to downclock to 0.44Ghz, before speeding back up again once GHelper is started up.

Laptop Model: Tuf F15 FX507ZC4

CPU: i5 12500H
PL1: 45W
PL2: 45W

Notes: it's either I disable Power Limits, or Restart laptop for it to work properly. Downclock usually lasts for a minute at most.

![20241219_180900](https://github.com/user-attachments/assets/b7e28c57-10cb-4b9f-bdb7-73eaf14fb1d0)

**Comment by seerge** [comment]
@Doughkey hello,

What specific settings are you setting ? Can you post a full log from the app as it's [asked in the issue template](https://github.com/seerge/g-helper/issues/new?assignees=&labels=&projects=&template=bug_report.yml) ?

How do absolutely identical settings work when you set them in Armoury ?

**Reply by Doughkey** [reply]
I'm actually running a clean installed Windows 11 so there's no Armory Crate. Like, after setting up OOBE, I updated the drivers and installed GHelper

Will post the Logs here too.

**Reply by TicherWithoutTitle** [reply]
hi, sorry for bringing up an old topic, but i have the exact same laptop as this guy, and i'm actually having the same problem. for 5 minutes clocked i had 0.45GHz.
ataching the log.
answering the question about armoury, i encounter a similar problem, but decided to uninstall all, so can't check.
this problem happend while using battery as well as using the charger
[log.txt](https://github.com/user-attachments/files/20465154/log.txt)

**Reply by seerge** [reply]
@TicherWithoutTitle if you experience same problems no matter what app do you use, it's most likely some underlying hardware issue causing that.

Usually if clock goes so low - it means that CPU is not getting enough power. Could be broken power supply or some element on the motherboard (whatever is responsible for the power delivery)

**Reply by TicherWithoutTitle** [reply]
ouh, okey then. thanks for this amazing app
